tests/unit/components/shared/WorldCountries.spec.js

```diff
@@ -0,0 +1,52 @@
+import WorldCountries from '@/components/shared/WorldCountries.vue';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import appInit from '../../testutils/appInit';
+import Vuex from 'vuex';
+
+const args = appInit(createLocalVue());
+const data = {
+    countries:{'SD':'SD'}
+};
+describe('WorldCountries.vue', () => {
+    let store;
+    let wrapper
+    beforeAll(() => {
+        store = new Vuex.Store({
+            modules: {
+                homeStore:{
+                    getters:{
+                        getMaxScoreMap: () => () => 0,
+                    }
+                }
+            },
+        });
```

Comment:
Don't think, it's useful. The component don't use the store

Reply:
Removed store.